public/app/features/search/page/components/RootFolderView.tsx

```diff
@@ -33,34 +49,23 @@ export const FolderView = ({
   const styles = useStyles2(getStyles);
 
   const results = useAsync(async () => {
-    const folders: DashboardViewItem[] = [];
+    const folders = await getChildren();
 
     if (!hidePseudoFolders) {
       if (contextSrv.isSignedIn) {
         const stars = await getBackendSrv().get('api/user/stars');
         if (stars.length > 0) {
-          folders.push({ title: 'Starred', icon: 'star', kind: 'folder', uid: '__starred', itemsUIDs: stars });
+          folders.unshift({ title: 'Starred', icon: 'star', kind: 'folder', uid: '__starred', itemsUIDs: stars });
```

Comment:
I _think_ this only matters for the old search page, so topnav turned off, but we need to reverse these unshifts though, otherwise we end up with

General
Recent
Starred

instead of the other way around like it used to be.